### ISSUE TITLE
Add help to write a config to update both IPV4 and IPV6 adresses.

### ISSUE THREAD

**Issue by EricV** [ddclient/ddclient#632]
Reading the last commit, I understand that work as been done already to better support IPV6. But I still do not catch how to update both IPV4 and IPV6 at the same time.

The config for infomaniak working for IPV4 is:

protocol=dyndns2 \
use=web, web=ipify-ipv4 \
server=infomaniak.com \
...

I can setup one for IPV6 replacingipify-ipv4 with  ipify-ipv6 :
protocol=dyndns2 \
use=web, web=ipify-ipv6 \
server=infomaniak.com \
...

But I would like to setup both at the same time. What should I do.

**Comment by LenardHess** [comment]
Note: Simultaneous IPv4/IPv6 support for `provider=dyndns2` got added in v3.11.0 so make sure you're on at least that version.

To write a config for both IPv4 and IPv6, you need to:
- Use `usev4`/`usev6` parameters instead of `use`
- Use `webv4`/`webv6` parameters instead of `web`

**Comment by EricV** [comment]
Thanks for responding. I use the git version so no problem.  I'm sorry to say that your answer is still not clear for me. Would you mind writing  three examples passed on the config extract I provided above

1. The Ipv4 only stuff,
2. The Ipv6 only stuff,
3. The combined Ipv4 + Ipv6 stuff,

Thanks for your patience.

**Comment by muru** [comment]
FWIW, if a CloudFlare example will help, here's what mine looks like:

```
protocol=cloudflare
zone=muru.dev
ttl=60
login=token
password=xxx

usev4=cmdv4, cmdv4='/usr/bin/external-ip' foo.muru.example.com
usev6=ifv6, ifv6='eth0' foo.muru.example.com
```
Aside: With CloudFlare, I had to create `A`/`AAAA` records for the subdomain first. Initially I only had the `A` record created, and `ddclient` wouldn't update a non-existent record, so only the `A` would update. Once I created both, `ddclient` started updating both.

So in your case, I'd imagine something like this should work:

```
protocol=dyndns2
server=infomaniak.com
...
usev4=web, web=ipify-ipv4 some-domain.ericv.example.com
usev6=web, web=ipify-ipv6 some-domain.ericv.example.com
```

**Comment by EricV** [comment]
Thanks for the suggestion. Will try.

**Comment by yodatak** [comment]
With latested ddclient 4.0 for infomaniak this config works

protocol=dyndns2
    ssl=yes
    server=infomaniak.com
    login=MYLOGIN
    password=XXXXX
    redirect=2
    MYDOMAIN.ORG
    usev4=webv4, web=https://ipv4.infomaniak.com/ip.php  MYDOMAIN.ORG
    usev6=webv6, web=https://ipv6.infomaniak.com/ip.php  MYDOMAIN.ORG

**Comment by riedinger** [comment]
I configured:

protocol=dyndns2
server=svc.joker.com
usev4=cmdv4,cmdv4='/usr/local/etc/get-my-ipv4-ip' XXX.eu
usev6=cmdv6,cmdv6='/usr/local/etc/get-my-ipv6-ip' XXX.eu

My problem is, that ddclient tries to update my ipv4 and ipv6 addresses within the same update request, which is not accepted by svc.joker.com. It would be necessary to split it up into two different update requests for ipv4 and ipv6.

The debug output shows:
DEBUG:   [dyndns2][XXX.eu][HTTP request][REQUEST]  url="https://svc.joker.com/nic/update?hostname=XXX.eu&myip=84.164.54.XYZ,2003:ca:df2f:7201:3803:4df1:672a:XYZ"

**Comment by Techwolf** [comment]
I have been waiting for a fix for this as this affects me also. Is there any workaround to send SEPARATE update for ipv4 and ipv6?

**Comment by njohnson-oss** [comment]
I requested Infomaniak to fix this issue on their end, so that IPv4 & IPv6 changes can be realized in a single request. They informed me today that the fix has been deployed. I tested it on ddclient with IPv4 and IPv6 lines, with success.

**Comment by nbarrientos** [comment]
Yup I was on the same boat and the fix they rolled out today allows again to submit changes for both IPv4 and IPv6 in the same request :)

**Comment by beadon** [comment]
there has been success on this issue, closing.

**Comment by Techwolf** [comment]
Still does not work for joker.com

**Comment by beadon** [comment]
For joker.com users hitting the combined IPv4+IPv6 request problem: joker.com uses the standard `dyndns2` protocol but rejects a single update request that contains both an IPv4 and IPv6 address (comma-separated in `myip`).

The fix is to use **two separate stanzas** in your `ddclient.conf` — one for IPv4 and one for IPv6 — each disabling the other address family. Joker.com conveniently provides dedicated IP-detection endpoints for each:

```
## Joker.com — IPv4
protocol=dyndns2,
server=svc.joker.com,
login=my-joker-dyndns-username,
password=my-joker-dyndns-password,
usev4=webv4, webv4=ipv4.svc.joker.com/nic/myip
usev6=disabled
your.domain.com

## Joker.com — IPv6
protocol=dyndns2,
server=svc.joker.com,
login=my-joker-dyndns-username,
password=my-joker-dyndns-password,
usev4=disabled,
usev6=webv6, webv6=ipv6.svc.joker.com/nic/myip
your.domain.com
```

The DynDNS `login` and `password` are the credentials Joker.com generates when you create a Dynamic DNS record — they differ from your main Joker.com account credentials.

This example has been added to [`ddclient.conf.in`](https://github.com/beadon/ddclient/blob/doc/joker/ddclient.conf.in) on the `doc/joker` branch in the beadon/ddclient fork, pending review for inclusion upstream.
